tests/integration_tests/test_streaming.py

```diff
@@ -15,20 +17,18 @@
 from guardrails.validators import LowerCase
 
 # Set mock OpenAI API key
-# os.environ["OPENAI_API_KEY"] = "sk-xxxxxxxxxxxxxx"
+os.environ["OPENAI_API_KEY"] = "sk-xxxxxxxxxxxxxx"
```

Comment:
Should set this via a mock instead of directly on the environment, otherwise this gets propagated to all tests that run after it.